packages/webapp/src/pages/Logs/Operation/Message/Show.tsx

```diff
@@ -50,6 +61,15 @@ export const ShowMessage: React.FC<{ message: MessageRow }> = ({ message }) => {
                     <div className="flex">
                         <LevelTag level={message.level} />
                     </div>
+                    {message.durationMs ? (
```

Comment:
it's checking the wrong value (duration)

Reply:
`duration` is really just a format. `durationMs` indicates if it should be displayed.